src/concepts/sql_optimization.md

```diff
@@ -15,6 +15,10 @@ take advantage of those two pieces.
 
 † Partitioned columns can be identified in the Schema Explorer in [re:dash](https://sql.telemetry.mozilla.org).
   They are the first few columns under a table name, and their name is preceded by a `[P]`.
```

Comment:
Is the `[P]` thing still applicable?  I'm not seeing it in redash

Reply:
I think this is only supported for Parquet. I will mention that here.